numpy/_core/tests/test_numeric.py

```diff
@@ -4117,7 +4117,7 @@ def test_broadcast_error_kwargs(self):
         #gh-13455
         arrs = [np.empty((5, 6, 7))]
         mit = np.broadcast(*arrs)
-        mit2 = np.broadcast(*arrs, **{})
+        mit2 = np.broadcast(*arrs)
```

Comment:
it kinda defeats the purpose of this test now

Reply:
🤔 I thought `**{}` is handled by Python, so `np.broadcast` wouldn't see the difference.
```
>>> def function(*args):
...   print("I was called with", len(args), "arguments:", args)
... 
>>> arrs = [1, 2, 3]
>>> function(*arrs)
I was called with 3 arguments: (1, 2, 3)
>>> function(*arrs, **{})
I was called with 3 arguments: (1, 2, 3)
>>> 
```

Reply:
Seems we should keep that unless `**{}` is the default. The reason is that the function is a C function. Actually, it looks like an object defined in C (`PyArrayMultiIter_Type`).

Reply:
It is a very old function, so maybe this doesn't apply anymore.